configs/config.clevo_ns70pu

```diff
@@ -0,0 +1,31 @@
+CONFIG_CCACHE=y
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_NO_STAGE_CACHE=y
+CONFIG_VENDOR_CLEVO=y
+CONFIG_MAINBOARD_VERSION="darp8"
+CONFIG_CBFS_SIZE=0xA00000
+# CONFIG_CONSOLE_SERIAL is not set
+# CONFIG_POST_IO is not set
+CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="Darter Pro"
```

Comment:
Needs to be changed based on SMBIOS values from the laptop running vendor firmware

Reply:
Fixed.